src/components/schedule-page/schedule-editing.component.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Button } from "../common-components";
+import ValidationDrawerComponent from "./validation-drawer/validation-drawer.component";
+import backend from "../../api/backend";
+import { ScheduleErrorActionType } from "../../state/reducers/schedule-errors.reducer";
+import { ActionModel } from "../../state/models/action.model";
+import { ScheduleError } from "../../common-models/schedule-error.model";
+import { useDispatch, useSelector } from "react-redux";
+import { ApplicationStateModel } from "../../state/models/application-state.model";
+import { ScheduleComponent } from "./table/schedule/schedule.component";
+import UndoIcon from "@material-ui/icons/Undo";
+import RedoIcon from "@material-ui/icons/Redo";
+
+export function ScheduleEditingComponent(): JSX.Element {
```

Comment:
Wolałbym, żeby nie nazywać komponentów - Component, może zamiast tego użyć ScheduleEditModeToolbar?

Reply:
`Toolbar` raczej nie, ponieważ ten component - to cała strona do edycji: w 68 linii on ma też `Schedule`. 
Ale myślę,że to potrzeba by było rozdzielić na dwa komponenty: toolbar, i strona do edycji ( komponent, który zawiera zarówno `toolbar` jak i `Schedule`.

Reply:
A faktycznie moje przeoczenie. Zgadzam się z tym co piszesz. Z tym, że musi to być w tym PR.